### PR TITLE
update rule postfix_network_listening_disabled

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- fix oval
- add Bash remediation
- add tests

#### Rationale:

Anssi effort

### CODE REVIEW

File: linux_os/guide/services/mail/postfix_client/postfix_network_listening_disabled/oval/shared.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
   <ind:textfilecontent54_test id="test_postfix_network_listening_disabled" check="all" check_existence="at_least_one_exists" comment="inet_interfaces in /etc/postfix/main.cf should be set correctly" version="1">
     <ind:object object_ref="obj_postfix_network_listening_disabled" />
+    <ind:state state_ref="state_postfix_network_listening_disabled" />
```
Comment (yuumasato) [inline linux_os/guide/services/mail/postfix_client/postfix_network_listening_disabled/oval/shared.xml:14]: What? Interesting, hehe.

File: linux_os/guide/services/mail/postfix_client/postfix_network_listening_disabled/bash/shared.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+CONFIGFILE="/etc/postfix/main.cf"
+
+if [ -f $CONFIGFILE ]; then
+  if grep -q '^[\s]*inet_interfaces.*$' $CONFIGFILE; then
+    sed -i "s/^\([[:space:]]*inet_interfaces\).*$/\1 = $var_postfix_inet_interfaces/" $CONFIGFILE
+  else
+    echo "inet_interfaces = $var_postfix_inet_interfaces" > $CONFIGFILE
+  fi
+fi
```
Comment (yuumasato) [inline linux_os/guide/services/mail/postfix_client/postfix_network_listening_disabled/bash/shared.sh:16]: Out of curiosity, as the Ansible corresponding remediation is implemented via `ansible_lineinfile`, is there a corresponding Bash remediation that could be used?

Comment (vojtapolasek) [reply linux_os/guide/services/mail/postfix_client/postfix_network_listening_disabled/bash/shared.sh]: Indeed, we have. Thanks for reminder.

**Review by yuumasato — APPROVED** [review]

**Pull request merged**